33549 text wrap style values (#34036)
mdn/content: files/en-us/web/css/text-wrap-style/index.md

### Values

- `auto`

>   - : Text is wrapped in the same way that `{{CSSXRef("text-wrap-mode")}}: wrap;` does. It does so in the most performant way for the browser and does not take into account the number of characters.

Reviewer: ```suggestion
  - : Text is wrapped in the most performant way for the browser and does not take into account the number of characters.
```

- `balance`

>   - : Text is wrapped in the same way that `text-wrap-mode: wrap;` does. The text is wrapped in a way that best balances the number of characters on each line, enhancing layout quality and legibility. Because counting characters and balancing them across multiple lines is computationally expensive, this value is only supported for blocks of text spanning a limited number of lines (six or less for Chromium and ten or less for Firefox).

Reviewer: ```suggestion
  - : Text is wrapped in a way that best balances the number of characters on each line, enhancing layout quality and legibility. Because counting characters and balancing them across multiple lines is computationally expensive, this value is only supported for blocks of text spanning a limited number of lines (six or less for Chromium and ten or less for Firefox).
```

- `pretty`

>   - : Text is wrapped in the same way that `text-wrap-mode: wrap;` does, except that the user agent will use a slower algorithm that favors better layout over speed. This is intended for body copy where good typography is favored over performance (for example, when the number of [orphans](/en-US/docs/Web/CSS/orphans) should be kept to a minimum).

Reviewer: ```suggestion
  - : Text is wrapped using a slower algorithm that favors better layout over speed. This is intended for body copy where good typography is favored over performance (for example, when the number of [orphans](/en-US/docs/Web/CSS/orphans) should be kept to a minimum).
```

- `stable`

>   - : Text is wrapped in the same way that `text-wrap-mode: wrap;` does, except that when the user is editing the content, the lines that come before the lines they are editing remain static rather than the whole block of text re-wrapping.

Reviewer: ```suggestion
  - : Text is wrapped such that when the user is editing the content, the lines that come before the lines they are editing remain static rather than the whole block of text re-wrapping.
```
